PermissionsUtilsUI/build.gradle

```diff
@@ -34,9 +34,11 @@ dependencies {
     implementation 'com.android.support:preference-v14:28.0.0'
     //noinspection GradleCompatible
     implementation 'com.android.support:recyclerview-v7:28.0.0'
+    implementation "org.jetbrains.kotlin:kotlin-reflect:$kotlin_version"
```

Comment:
@mohamed-garci Is that mandatory ?

Reply:
Full reflection requires the kotlin-reflect library in addition to kotlin-stdlib.

Reply:
@mohamed-garci Sorry I wasn't enough specific : can you tell me where reflection is used in the feature and why is it necessary ? Thanks !

Reply:
@Steven-DriveQuant 
reflection is used in `RequestPermissionActivity` line 138

It's not necessary, but it's very helpful in this case.

Reply:
Remove the dependency to avoid integration conflicts

Reply:
Done